infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/EventLogger.java

```diff
@@ -273,11 +273,15 @@ public void terminalPowBlockTtdEta(final UInt256 ttd, final Duration eta, final
   }
 
   public void lateBlockImport(
-      final Bytes32 root, final UInt64 slot, final UInt64 proposer, final String timings) {
-    String slowBlockLog =
+      final Bytes32 root,
+      final UInt64 slot,
+      final UInt64 proposer,
+      final String timings,
+      final String result) {
+    final String slowBlockLog =
         String.format(
-            "Late Block Import *** Block: %s proposer %s %s",
-            LogFormatter.formatBlock(slot, root), proposer, timings);
+            "Late Block Import *** Block: %s proposer %s - %s - result: %s",
```

Comment:
Just a bit of a nit but think  "Late Block Import *** Block: %s Proposer: %s Result: %s %s" is cleaner and more consistent

And swap result and timings